backend/config/config.go

```diff
@@ -256,10 +256,12 @@ func NewConfig(appConfigFilename string, accountsConfigFilename string) (*Config
 	}
 	config.load()
 	appconf := config.appConfig
+	accountconf := config.accountsConfig
 	migrateFiatList(&appconf)
 	migrateFiatCode(&appconf)
 	migrateElectrumX(&appconf)
 	migrateUserLanguage(&appconf)
+	migrateActiveTokens(&accountconf)
 	if err := config.SetAppConfig(appconf); err != nil {
 		return nil, errp.WithStack(err)
 	}
```

Comment:
There is SetAppConfig() but no corresponding SetAccontsConfig(), so it is confusing what is happening and if the migration is actually saved. It does not look like the migration is actually saved right now.

You probably want to use `config.ModifyAccountsConfig(...)` instead and put `migrateActiveTokens()` inside the callback of that function.

Reply:
Sorry no idea how I missed that, I remember looking and thinking _ah its persisted here_ but that was only the appconf..